exchanges/gateio/gateio.go

```diff
@@ -1345,6 +1345,27 @@ func (g *Gateio) GetUsersTotalBalance(ctx context.Context, ccy currency.Code) (*
 	return response, g.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, walletTotalBalanceEPL, http.MethodGet, walletTotalBalance, params, nil, &response)
 }
 
+// ConvertSmallBalances converts small balances to Gate IO token (GT). This can only be performed once a day.
```

Comment:
> You can convert small balances of supported crypto (excluding delisted crypto) above 0.00000001 GT and below 0.0001 BTC in value to GT. You can make a conversion every 6 hours. A quota of 100 GT is available per account every 3 days.

I'm not seeing your comment in gateio docs, so the above was taken from frontend

Reply:
Was going off from what errors were returned but if that is what docs is, then this must have changed. 👍

Reply:
I got a super vague error when I tried it twice in a row, so I went looking

Reply:
It also times out intermittently, just doesn't want to respond. 😆 🤷